bouncer/shared/setup_boost_pools.ts

```diff
@@ -1,19 +1,78 @@
-import { ingressEgressPalletForChain, getAssetsForChain, Asset } from '../shared/utils';
+import { InternalAssets as Assets } from '@chainflip/cli';
+import {
+  ingressEgressPalletForChain,
+  getAssetsForChain,
+  chainFromAsset,
+  Asset,
+} from '../shared/utils';
 import { submitGovernanceExtrinsic } from '../shared/cf_governance';
 import { observeEvent } from './utils/substrate';
+import { addBoostFunds } from './boost';
 
-type BoostPoolId = {
+export type BoostPoolId = {
   asset: Asset;
   tier: number;
 };
 
 // These are the tiers of boost pools that will be created for each asset
 const boostPoolTiers = [5, 10, 30];
 const chains = ['Ethereum', 'Polkadot', 'Bitcoin', 'Arbitrum'] as const;
+const fundBtcBoostPoolsAmount = 2; // Put 2 BTC in each Btc boost pool after creation
 
+/// Submits a single governance extrinsic that creates the boost pools for the given assets and tiers.
+/// All assets must be be from the same chain.
+export async function createBoostPools(newPools: BoostPoolId[]): Promise<void> {
+  if (newPools.length === 0) {
+    throw new Error('No boost pools to create');
+  }
+  const chain = chainFromAsset(newPools[0].asset);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const observeBoostPoolEvents: Promise<any>[] = [];
+
+  for (const pool of newPools) {
+    if (chainFromAsset(pool.asset) !== chain) {
+      throw new Error(`All assets must be from the same chain`);
+    }
+
+    if (pool.tier <= 0) {
+      throw new Error(`Invalid tier value: ${pool.tier}`);
```

Comment:
No reason not to be explicit
```suggestion
      throw new Error(`Tier value: ${pool.tier} cannot be less than 0`);
```